requirements.txt

```diff
@@ -4,23 +4,29 @@
 #
 #    pip-compile
 #
+archivessnake==0.9.1
+    # via -r requirements.in
 attrs==21.4.0
-    # via jsonschema
+    # via archivessnake
+boltons==21.0.0
+    # via archivessnake
 certifi==2021.10.8
     # via requests
 charset-normalizer==2.0.11
     # via requests
 idna==3.3
     # via requests
-importlib-resources==5.4.0
-    # via jsonschema
-jsonschema==4.4.0
-    # via -r requirements.in
-pyrsistent==0.18.1
-    # via jsonschema
+more-itertools==8.12.0
+    # via archivessnake
+pyyaml==6.0
+    # via archivessnake
+rapidfuzz==2.0.5
+    # via archivessnake
 requests==2.27.1
```

Comment:
Same with these changes to dependencies - let's add jsonschema back and re-generate requirements.txt.